lib/jekyll/converters/smartypants.rb

```diff
@@ -29,7 +29,12 @@ def output_ext(_)
       end
 
       def convert(content)
-        Kramdown::Document.new(content, @config).to_html.chomp
+        document = Kramdown::Document.new(content, @config)
+        html_output = document.to_html.chomp
+        document.warnings.each do |warning|
+          Jekyll.logger.warn "Kramdown warning:", warning
```

Comment:
It looks like there are currently no (legitimate) warnings raised by any of the parsers SmartyPants is using in kramdown. Perhaps we just rip this out?

Reply:
Apologies for the late response. No objections to remove it, if it doesn't help in any way